### PR TITLE
Übersetzung hinzugefügt

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: locale/afrux-theme-base.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      grid:
+        columns:
+          avatar:
+            title: Avatar
+
+    settings:
+      edit_footer_links: Bearbeite Footer
+      footer_description: Footer Beschreibung
+      footer_bottom_line: Footer Endeffekt
+      hero_banner: Willkommen Hero Bild
```
Comment (Kakifrucht) [inline locale/afrux-theme-base.yml:25]: I guess we must keep this, I'm not sure how to translate "Hero" here. I don't even know why Flarum uses that word in the first place.

File: locale/afrux-theme-base.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      hero_banner_position: "Hero Bild CSS Position (beispiel: 'center top' oder 'center 20%')"
```
Comment (Kakifrucht) [inline locale/afrux-theme-base.yml:26]: "Beispiel" uppercased

File: locale/afrux-theme-base.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      links_modal:
+        add_group: Neue Linkgruppe
+        group_name: Gruppen {number} Name
```
Comment (Kakifrucht) [inline locale/afrux-theme-base.yml:29]: "Gruppe" singular, since it will be replaced like "Gruppe 1 Name" etc.

**Review by Kakifrucht — CHANGES_REQUESTED** [review]

File: locale/afrux-theme-base.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+afrux-theme-base:
+  admin:
+    extensions:
+      extensions: Erweiterungen
+      title: Erweiterungen
+      desc: Liste aller installierten Erweiterungen.
+    extension:
+      sections:
+        content: Basis Einstellungen
```
[the remark this answers is not in the document]
Comment (Justman10000) [reply locale/afrux-theme-base.yml:9]: Right!

File: locale/afrux-theme-base.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+afrux-theme-base:
+  admin:
+    extensions:
+      extensions: Erweiterungen
```
[the remark this answers is not in the document]
Comment (Justman10000) [reply locale/afrux-theme-base.yml:4]: I know! I wait for [my pulls](https://github.com/afrux/flarum-theme-base/pulls) to be accepted...

File: locale/afrux-theme-base.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      go_back: Zurück zum Forum
+
+    users:
+      grid:
+        columns:
+          avatar:
+            title: Avatar
+
+    settings:
+      edit_footer_links: Bearbeite Footer
```
[the remark this answers is not in the document]
Comment (Justman10000) [reply locale/afrux-theme-base.yml:22]: Dunno, would take links to that...

File: locale/afrux-theme-base.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (Kakifrucht) [reply locale/afrux-theme-base.yml:4]: In that case we'll wait until that is resolved. Please ping me when that is the case.

File: locale/afrux-theme-base.yml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+      footer_description: Footer Beschreibung
+      footer_bottom_line: Footer Endeffekt
```
[the remark this answers is not in the document]
Comment (Justman10000) [reply locale/afrux-theme-base.yml:24]: How else do you translate?

**Pull request closed without merging**